src/parse/decl.rs

```diff
@@ -443,8 +443,10 @@ impl<I: Iterator<Item = Lexeme>> Parser<I> {
             Some(ctype) => ctype,
             None => {
                 if signed.is_none() {
-                    // if it's not an id, it's invalid anyway
-                    // other parts of the parser will have a better error message
+                    // these are the only tokens that can come before an id in a
+                    // declarator if it's not a declarator, it's invalid anyway
```

Comment:
You line wrapped it and it changed the meaning :(
```suggestion
                    // these are the only tokens that can come before an id in a declarator
                    // if it's not a declarator, it's invalid anyway
```

Reply:
oh no! hecc